CODEOWNERS

```diff
@@ -14,3 +14,4 @@
 * @st96d045         # Esteban Serrano
 * @ats0stv          # Arun Thundyill Saseendran
 * @RamXX            # Ramiro Salas
+* @farhan-pasha     # Farhan Pasha
```

Comment:
After merging this in; I’ll have to give Farhan write access to the repo.

Otherwise, it will remain invalid.